osu.Game.Rulesets.Taiko/Objects/Drawables/DrawableFlyingHit.cs

```diff
@@ -27,5 +27,11 @@ protected override void LoadComplete()
             base.LoadComplete();
             ApplyResult(r => r.Type = r.Judgement.MaxResult);
         }
+
+        protected override void LoadSamples()
+        {
+            // block base call - flying hits are not supposed to play samples
```

Comment:
It looks like this is going to stop the base(base) call of `LoadSamples`, causing it to never load any samples. This may not be an issue for taiko since the samples are played by the drum, but not sure how safe it is...

Reply:
Yep, it will. I also considered exposing `DrawableHit.updateSamplesFromTypeChange()` as protected and calling it manually in the flying hit ctor for it to write the proper value to prevent overwrite later, but that feels like a worse hack than this, and sort of dangerous seeing that on current `master` it's possible that a flying hit's `PlaySamples()` will get called (the above resolution doesn't cause double sample playback as it writes the samples to the hit object and not the *drawable* hit object, but it seems like it might do so at some point in the future).

Reply:
I actually think this is fine due to the comment being present here. Somehow glossed over reading it.

These are a special kind of object anyways.